src/js/main.js

```diff
@@ -129,43 +129,48 @@ function startMaqueeMarq() {
 const mouse = {
 	x: 0,
 	y: 0,
-	endX: 0,
-	endY: 0,
 	dragCallback: false, // What to do when a dragged element is moved
-	endCallback: false // What to do when a dragging stops
+	endCallback: false, // What to do when a dragging stops
+	executeCallback: function(e) {
+		if (mouse.dragCallback) {
+			if (e.cancelable) {
```

Comment:
What would be the exact benefit of this `e.cancelable`?

Reply:
It will only cancel when the event _is_ cancelable, so this code can be used for mouse (cancelable) and touch (non-cancelable) events.